pkg/workload/kv/kv.go

```diff
@@ -942,40 +965,90 @@ func (g *sequentialGenerator) state() string {
 	return fmt.Sprintf("S%d", g.seq.read())
 }
 
-type zipfGenerator struct {
-	seq    *sequence
+type writeSequenceSource struct {
+	*sequence
+}
+
+func (wss writeSequenceSource) Int63() int64 {
+	return int64(wss.Uint64() & 0x7fffffffffffffff) // Masks high bit
+}
+
+func (wss writeSequenceSource) Uint64() uint64 {
+	hasher := crc64.New(crc64.MakeTable(crc64.ECMA))
+	buf := []byte{0, 0, 0, 0, 0, 0, 0, 0}
+	// only increment the sequence during write calls
```

Comment:
Super nit: perhaps this comment would be better in the `wss.write` method? Was confused thinking this function was doing the incrementing when it appears to be baked into `write`.